### DISCUSSION TITLE
Does Fairy Stockfish ponder when the opponent plays?

### DISCUSSION THREAD

**Discussion by ARNAV69420** [fairy-stockfish/Fairy-Stockfish#612] [Q&A]
So, I was using this website https://fairyground.vercel.app/ to play with one of the variants. I had a doubt does the engine think during the time the opponent plays. If not is there any way to use that setting. Also, if its not possible in the website. Is this possible by using the CLI version

**Comment by ianfab** [comment]
Fairyground is extremely simplistic and does not support pondering. The native version supports pondering as specified in the respective protocol, e.g. UCI and CECP.
